src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -32,9 +32,6 @@ final class JsDelivrEsmResolver implements PackageResolverInterface
 
     public function __construct(
         HttpClientInterface $httpClient = null,
-        private readonly string $versionUrlPattern = self::URL_PATTERN_VERSION,
-        private readonly string $distUrlPattern = self::URL_PATTERN_DIST,
-        private readonly string $distUrlCssPattern = self::URL_PATTERN_DIST_CSS
```

Comment:
Unrelated: but it really doesn't make sense to have these URLs configurable. We are hitting very exact URLs.

Reply:
typo, should be @tabler/core https://www.jsdelivr.com/package/npm/@tabler/core